### PR TITLE
LPS-169456 ObjectDefintion The PUT method should only require the same fields as the POST method

### PR DESCRIPTION
**Pull request by locpham97**
Hi team,
This PR is for ticket https://issues.liferay.com/browse/LPS-169456. Please help me to review it.
Thanks a lot.
cc:@taiphamlr

### CODE REVIEW

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 1]
```diff
@@ -346,7 +346,9 @@ public ObjectDefinition putObjectDefinition(
 				titleObjectFieldId,
 				GetterUtil.getBoolean(
 					objectDefinition.getAccountEntryRestricted()),
-				GetterUtil.getBoolean(objectDefinition.getActive(), true),
+				GetterUtil.getBoolean(
+					objectDefinition.getActive(),
+					serviceBuilderObjectDefinition.getActive()),
```
Comment (carolmariaabb) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:351]: @locpham97 @TaiPhamLR why did you add the current value of the `serviceBuilderObjectDefinition` here, but you didn't add it below in `GetterUtil.getBoolean(objectDefinition.getPortlet())`?

Comment (TaiPhamLR) [reply modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:351]: Hi @carolmariaabb. Before changing these line codes (349-351), an exception was thrown at [ObjectDefinitionLocalServiceImpl#_validateActive](https://github.com/liferay-objects/liferay-portal/blob/9758d79ea9edcfa3be635da813bcc1aa54b74b9d/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java#L1337) if 'active' was not set in request json. I use the 'active' value of serviceBuilderObjectDefinition (it's an old value in the database) in case the input json does not include 'active'.
If I don't understand properly, please let me know.
Thanks a lot.

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
@TaiPhamLR @locpham97 feel free to send this PR directly to Brian. The failures are not related to your changes. Thanks!

**Comment by naraandrade** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/127868
